sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/GoneAndRetryWithRetryPolicy.java

```diff
@@ -115,8 +115,8 @@ private CosmosException logAndWrapExceptionWithLastRetryWithException(Exception
             String exceptionType;
             if (exception instanceof GoneException) {
                 exceptionType = "GoneException";
-            } else if (exception instanceof PartitionKeyRangeGoneException) {
-                exceptionType = "PartitionKeyRangeGoneException";
+            } else if (exception instanceof PartitionIsMigratingException) {
```

Comment:
PartitionKeyRangeGoneException is one of the noRetriable exception, so by theory we will never reach here.

And I think what we meant here probably is PartitionIsMigratingException? Please correct me if I am wrong.